modules/messages/tests/server/message-to-influx.server.service.tests.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var mongoose = require('mongoose'),
+    path = require('path'),
+    async = require('async'),
+    User = mongoose.model('User'),
+    Message = mongoose.model('Message'),
+    messageToInfluxService =
+  require(path.resolve('./modules/messages/server/services/message-to-influx.server.service'));
+require('should');
+
+/**
+ * Unit tests
+ */
+describe('Message to influx server service Unit Tests:', function() {
+  var user1,
+      user2;
+
+  // here we create the users
```

Comment:
It's the same as elsewhere at the codebase and following the example from Meanjs boilerplate.

Each test cleans up after itself and even if you'd "pull the plug" intentionally in the middle of it, on next run it would empty it again. CI tests on Travis run on a fresh container every time anyway.

The same issue would come up with e2e tests (which we don't have) and database model tests (which we have), so mocking this just for unit testing isn't really worth it.